_posts/2021-09-02-rocket-validator.md

```diff
@@ -0,0 +1,113 @@
+
+HTML is the basis of web pages, but still a perfectly valid HTML page can have accessibility issues. For example, a web page may have a bad combination of colors that results in low contrast: a light color text over a light background may lead to unreadable text by people with low vision. If a video has no captions, deaf users won't understand what it's talking about. Sections of the document may be not accessible for users that don't employ a mouse to move through the document. Zooming and scaling may be disabled, making it impossible for the users to enlarge the web page to see it better. Many known issues are easily detected using automated accessibility checkers like [axe-core](https://github.com/dequelabs/axe-core).
+
+## The power of automated site validation
+
+So, it's great that we have tools that help in detecting issues on web pages. Both the W3C HTML validator and axe-core are open source and can be incorporated into our web development flow. But, there's an important limitation: they can only check one URL at a time. When you have a small website with just a few web pages, it might be an option to do manual checks using these tools. But, what can we do with a large site like Elixir School? According to the latest reports, Elixir School has more than 1,000 web pages. If we want to check them all with the W3C Validator and axe-core, we would need to run 2,000 manual checks!
+
+## 🚀 Rocket Validator to the rescue!
+
+[Rocket Validator](https://rocketvalidator.com) is an online service that performs site-wide validation of large websites. It's basically an automated web crawler that will find the internal web pages of a site and validate each web page found using the W3C Validator and axe-core, producing detailed reports on the issues found, within seconds.
+
+This service, also powered by Elixir, Phoenix, LiveView and Oban, is a subscription-based service for web agencies and freelance developers. But, it's free for open source projects like [Elixir School](https://github.com/elixirschool/school_house/pulls?q=is%3Apr+label%3A%22rocket+validator%22+is%3Aclosed) and [Docusaurus](https://github.com/facebook/docusaurus/pulls?q=is%3Apr+rocket+validator)! We're using Rocket Validator to run reports on the new Elixir School and to monitor the site via weekly-scheduled and deploy-triggered validations.
+
+## Running a site validation report
+
+To run a site-wide validation report, we just need to enter a starting URL. This is usually the front page of the site we're going to validate, but it can be any internal URL, or an XML or TXT sitemap containing the web pages to validate. Rocket Validator will visit this initial URL and discover the internal web pages, adding them to the report.
```

Comment:
```suggestion
To run a site-wide validation report, we need to enter a starting URL. This URI is usually the front page of the site we're going to validate, but it can be any internal URL, or an XML or TXT sitemap containing the web pages to validate. Rocket Validator will visit this initial URL and discover the internal web pages, adding them to the report.
```